### PR TITLE
docs: add a mechanical loop example

### PR DESCRIPTION
**Pull request by gaurivn**
**Description**
Added a mechanical loop example in [this page](https://cyberbotics.com/doc/reference/solidreference).

**Related Issues**
This pull-request fixes issue #1818 

**Tasks**
Add the list of tasks of this PR.
  - [x] Add a mechanical loop example
  - [ ] Task 2

**Documentation**
If this pull-request changes the doc, add the link to the related page, including the `?version=BRANCH_NAME`, such as:
https://cyberbotics.com/doc/guide/getting-started-with-webots?version=develop

https://cyberbotics.com/doc/reference/solidreference?version=master

**Screenshots**
If this pull-request includes any new robots/simulations/etc. add one or more screenshots of the result.

**Additional context**
Add any other context about the pull-request here.

### CODE REVIEW

File: docs/reference/solidreference.md [hunk 1]
```diff
@@ -16,3 +16,16 @@ The only constraint when referring to a [Solid](solid.md) is that both [Solid](s
...
+
+```
+ DEF FRONT_AXLE_HINGE HingeJoint {
+   jointParameters HingeJointParameters {
+     anchor -0.08 -0.038 -0.11
+   }
+   endPoint SolidReference {
+     solidName "front axle"
+   }
+ }
```
Comment (DavidMansolino) [inline docs/reference/solidreference.md:30]:
```suggestion
HingeJoint {
  jointParameters HingeJointParameters {
    anchor 0 0 1
  }
  endPoint Solid {
    name "front axle"
  }
}
HingeJoint {
  jointParameters HingeJointParameters {
    anchor 0 0 -1
  }
  endPoint SolidReference {
    solidName "front axle"
  }
}
```

For a more complete example, it is better to show the 'referred' solid in the example.

File: docs/reference/solidreference.md [hunk 2]
```diff
@@ -16,3 +16,16 @@ The only constraint when referring to a [Solid](solid.md) is that both [Solid](s
 
 - `solidName`: This field specifies either the static environment (if the value is `<static environment>`) or the name of an existing [Solid](solid.md) node to be linked with the [Joint](joint.md)'s closest upper [Solid](solid.md) node.
 Referring to the [Joint](joint.md) closest upper [Solid](solid.md) node or to a [Solid](solid.md) node which has no common upper [Solid](solid.md) with the [Joint](joint.md) is prohibited.
+
+### Mechanical Loop Example
+
```
Comment (DavidMansolino) [inline docs/reference/solidreference.md:21]:
```suggestion

The [SolidReference](solidreference.md) node is particularly useful to create closed mechanical loops, here is a very simple example:

```

File: docs/reference/solidreference.md (hunk 1 above)
Comment (gaurivn) [reply docs/reference/solidreference.md]: Okay, thank you, I made changes.

**Comment by DavidMansolino** [comment]
@gaurivn thank you, feel free to merge the PR.

**Comment by gaurivn** [comment]
> @gaurivn thank you, feel free to merge the PR.

Welcome, I'll merge this PR

**Pull request merged**